index.d.ts

```diff
@@ -0,0 +1,73 @@
+import { type EventEmitter } from 'events'
+import type bunyan from 'bunyan'
+
+export as namespace hato;
+
+/**
+ * Plugins
+ */
```

Comment:
I think I will move these definitions to the plugin dir later. JFYI

Reply:
Ok, I think that's fine :+1: